internal/wrphandlers/mocktr181/handler.go

```diff
@@ -158,13 +162,24 @@ func (h Handler) get(names []string) (int64, []byte, error) {
 
 	for _, name := range names {
 		for _, mockParameter := range h.parameters {
-			if strings.HasPrefix(mockParameter.Name, name) {
+			if !strings.HasPrefix(mockParameter.Name, name) {
+				continue
+			}
+
+			switch mockParameter.Access {
+			case "r", "rw", "wr":
 				result.Parameters = append(result.Parameters, Parameter{
 					Name:       mockParameter.Name,
 					Value:      mockParameter.Value,
 					DataType:   mockParameter.DataType,
 					Attributes: mockParameter.Attributes,
 				})
+			default:
```

Comment:
For a non-readable parameter:
```
{
                "name": "Device.Bridging.Bridge.8.VLANNumberOfEntries",
                "value":        "1",
                "access":       "w",
                "type": 2
 }
```

This is the expected response:
{
  "message": "Invalid parameter name",
  "statusCode": 520
}